src/commands/channels/configure.ts

```diff
@@ -0,0 +1,178 @@
+
+export default class ChannelsConfigure extends MixCommand {
+  static description = `update channel details in a project
+
+Configure the modalities and color of an existing channel in a Mix project.
+If either value is not supplied, the existing property on the channel will be unmodified.
+
+A note on channel modes and colors:
+For your convenience, you may enter modes and colors case-insensitively.
+In particular, you can spell any given mode/color with lowercase or capital letters,
+and with dashes ('-') in place of underscores ('_'). The value will be internally converted
+into the format the Mix API expects before the request is made. As an example, the values
+'light-pink', 'light_Pink', and 'LIGHT-PINK' are all equivalent to Mix's 'LIGHT_PINK'.
+
+Acceptable channel modes are:
+audioscript
+dtmf
+interactivity
+richtext
+tts
+
+Acceptable channel colors are:
+blue
+brown
+corn-flower
+cyan
+green
+grey
+indigo
+light-green
+light-grey
+light-orange
+light-pink
+light-purple
+orange
+pink
+purple
+ruby
+salmon
+sky
+teal
+yellow
+
+  `
+
+  static examples = [
+    'mix channels:configure -P 1922  \\',
+    '  --channel bc40667c-e0f6-11ec-9d64-0242ac120003 \\',
+    '  --mode DTMF --mode TTS \\',
+    '  --color SALMON',
+  ]
+
+  static flags = {
+    project: MixFlags.projectFlag,
+    channel: flags.string({
+      char: 'C',
+      description: 'channel ID',
+      required: true,
+    }),
+    mode: {
+      ...MixFlags.modesFlag,
+      required: false,
+    },
+    color: flags.string({
+      description: 'channel color',
+    }),
+    // output flags
+    json: MixFlags.jsonFlag,
+    yaml: MixFlags.yamlFlag,
+  }
+
+  get domainOptions(): DomainOption[] {
+    debug('get domainOptions()')
+    return ['project']
+  }
+
+  tryDomainOptionsValidation(options: any, domainOptions: DomainOption[]) {
+    debug('tryDomainOptionsValidation()')
+    super.tryDomainOptionsValidation(options, domainOptions)
+
+    // At least one flag must be set
+    if (options.mode === undefined && options.color === undefined) {
+      this.error('At least one of --mode and --color must be set.')
+    }
+
+    if (options.color !== undefined) {
+      try {
+        validateChannelColor(options.color)
+      } catch {
+        throw (eInvalidValue(`Invalid color ${chalk.red(options.color)} supplied.`,
+          [
+            'Check value of --color flag and try again.',
+            `Enter ${chalk.green('mix channels:configure help')} to review valid color options.`,
+          ]))
+      }
+    }
+
+    if (options.mode !== undefined) {
+      validateChannelModeOptions(options.mode)
+    }
+
+    if (!(options.mode && options.color)) {
+      this.warn(chalk.yellow(`The V4 API currently requires both the --color and --mode flags to be set.
+This is due to a server-side error in which the default values for these flags (i.e., when not modified)
+are not recognized as valid.)`))
+    }
+  }
+
```

Comment:
Let's override `setRequestActionMessage` to say something like:

"Configuring channel _channelID_ in project _projectID_"